css/comments.css

```diff
@@ -0,0 +1,165 @@
+
+
+.typography ul.action-links, .typography ul.comments-list, .typography ul.no-bullet {
```

Comment:
The use of .typography is somewhat specific to the theme. Can we wrap the entire comments template in a div class something like 'comments' add on so we don't break exising themes that use .action-links or anything else generic.